docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,10 +170,32 @@ correct Google Workspace plan.
 
 ### Create an OAuth client ID
 
-In the [Create an OAuth client ID](https://console.cloud.google.com/apis/credentials/oauthclient) page of the GCP console, select "Web application" as the Application type, pick a name, then add the following as an authorized redirect URI.
+#### Determine your callback URL
+
+You'll need a callback URL in order to complete the creation of an OAuth client ID with Google. For
+most deployments, this URL is your `public_addr` value. This consists of a domain name and port number.
+
+You can lookup the `public_addr` of your Teleport proxy using the following command:
```

Comment:
```suggestion
You can lookup the `public_addr` of your Teleport proxy using the following command (replace `example.teleport.com` with the address of your cluster):
```

This is actually a bit of a chicken and egg problem because you need the public address of the cluster in order to get the public address of the cluster 😅

Reply:
Agreed, unless there's a case where the public address of the proxy is not the same as the cluster name (I don't think there is).

Since this guide includes the tctl partial early, I think it's a safe assumption that at this point in the guide they know the proxy address.

Reply:
There is, but it's far less common now that we're encouraging people to set their cluster name to an FQDN.

Reply:
I think the changes are meant to solve the lion's share of use cases, which I believe it does.